commands/gifs/animegif.js

```diff
@@ -1,33 +1,56 @@
 const fetch = require('node-fetch');
-const { tenorAPI } = require('../../config.json');
+const config = require('../../config.json');
 const { Command } = require('discord.js-commando');
+const { MessageEmbed } = require('discord.js');
 
-// Skips loading if not found in config.json
-if (!tenorAPI) return;
+if (!config.tenorAPI) return;

```

Comment:
Why are you importing the entire config and then you check if it has a tenorAPI property instead just importing tenorAPI and checking if its a truthy value? (exists)